students/kimeunji/westagram/users/views.py

```diff
@@ -38,15 +39,23 @@ def post(self, request):
             return JsonResponse({'message':'KEY_ERROR'}, status = 400)
         
 class LogInView(View):
-    def get(self, request):
+    def post(self, request):
         try:
             data     = json.loads(request.body)
             email    = data['email']
             password = data['password']
+            user     = User.objects.filter(email=email)
             
-            if not User.objects.filter(email = email).exists() or User.objects.get(email = email).password != password:
-                return JsonResponse({'message':'INVALID_USER'}, status = 401)
-            return JsonResponse({'message':'SUCCESS'}, status = 200)
-        
+            if not user.exists():
+                return JsonResponse({'message':'EMAIL_DOSE_NOT_EXISTS'}, status = 401)
```

Comment:
```suggestion
                return JsonResponse({'message':'INVALID_USER'}, status = 401)
```
보안상 이메일이 틀렸는지, 비밀번호가 틀렸는지 모르는 것이 좋습니다.